src/hmem_cuda.c

```diff
@@ -82,6 +87,12 @@ const char *ofi_cudaGetErrorString(cudaError_t error)
 	return cuda_ops.cudaGetErrorString(error);
 }
 
+CUresult ofi_cuPointerGetAttribute(void *data, CUpointer_attribute attribute,
```

Comment:
Have you come across any situation where we would need this outside of providers that want to handle GPU buffers without requiring FI_MR_HMEM?

Reply:
Yes. The CUDA memory monitor uses this function to get a buffer ID for a GPU address.

Reply:
Ah yes, we've talked about that. Looking at that PR next.